packages/valory/skills/market_creation_manager_abci/behaviours.py

```diff
@@ -1132,10 +1133,10 @@ def get_payload(self) -> Generator[None, None, str]:
 
     def get_markets(self) -> Generator[None, None, Tuple[List[Dict[str, Any]], int]]:
         """Collect FMPMM from subgraph."""
+        creator = self.synchronized_data.safe_contract_address.lower()
+        self.context.logger.info(f"get_markets {creator=}")
```

Comment:
```suggestion
        self.context.logger.debug(f"get_markets {creator=}")
```

Reply:
I will delete this line, as it's contained in other parts of the logs.